functions/Measure-DbaDiskSpaceRequirement.ps1

```diff
@@ -74,20 +74,32 @@ function Measure-DbaDiskSpaceRequirement {
         [DbaInstanceParameter]$Source,
         [Parameter(Mandatory = $true, ValueFromPipelineByPropertyName = $true)]
         [string]$Database,
+        [Parameter(Mandatory = $false, ValueFromPipelineByPropertyName = $true)]
         [PSCredential]$SourceSqlCredential,
         [Parameter(Mandatory = $true, ValueFromPipelineByPropertyName = $true)]
         [DbaInstanceParameter]$Destination,
         [Parameter(Mandatory = $false, ValueFromPipelineByPropertyName = $true)]
         [string]$DestinationDatabase,
+        [Parameter(Mandatory = $false, ValueFromPipelineByPropertyName = $true)]
         [PSCredential]$DestinationSqlCredential,
+        [Parameter(Mandatory = $false, ValueFromPipelineByPropertyName = $true)]
         [PSCredential]$Credential,
         [Alias('Silent')]
         [switch]$EnableException
     )
     begin {
         $local:cacheMP = @{}
         $local:cacheDP = @{}
-
+        function Get-MountPoint {
+            [CmdletBinding()]
+            param(
+                [Parameter(Mandatory = $true)]
+                $computerName,
+                [PSCredential]$credential
+            )
+            Get-WmiObject -Class Win32_MountPoint -ComputerName $computerName -Credential $credential |
```

Comment:
Thanks so much, @pollusb ! Can you please use `Get-DbaCmObject` instead of Get-WmiObject? Then we should be set 👍